mithril-signer/Dockerfile.ci

```diff
@@ -29,7 +34,7 @@ COPY mithril-signer/config /app/config
 
 # Workdir
 WORKDIR /app/
-RUN chown -R appuser /app/ && chmod a+x /app/bin/mithril-signer && chmod a+x /app/bin/cardano-cli
+RUN chown -R appuser /app/ && chmod a+x /app/bin/mithril-signer
```

Comment:
The chmod for the cardano-cli is not done anymore when `EMBED-CARDANO-CLI=1`

Reply:
The cardano-cli comes from a downloaded archive which already have the execution rights.
This chmod seems to be needed when we download locally a signer from an artifact. In that case permissions may be lost (https://github.com/input-output-hk/mithril/commit/cdfa8780e9d5d242b85937b292bc62b2aecc0c13).
If you think it's safer to add the chmod command, I propose to add it at the end of the instruction into the conditional directive (just after the 'rm cardano-bin.tar.gz).
What is your opinion ?